pkgs/applications/editors/emacs/sources.nix

```diff
@@ -75,6 +75,9 @@ in
     variant = "mainline";
     rev = "28.2";
     hash = "sha256-4oSLcUDR0MOEt53QOiZSVU8kPJ67GwugmBxdX3F15Ag=";
+    templateNativeCompDriverOptionsPatch = builtins.path {
+      path = ./native-comp-driver-options-28.patch;
```

Comment:
IIRC, the motivation of using `builtins.path` is to avoid rebuilds caused by renaming patches.  However, it you do not specify a fixed `name` for `builtins.path`, you cannot avoid those rebuilds.